src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -114,7 +115,7 @@ const OnDemandRadioPage = ({ pageData }) => {
         description={shortSynopsis}
         openGraphType="website"
       />
-
+      <LinkedData type="RadioChannel" seoTitle={headline} />
```

Comment:
Should we pass through datePublished and dateModified? @aeroplanejane 

What is the reasoning behind using "RadioChannel" over "AudioObject"?

Reply:
pass through datePublished and dateModified for what? Don't understand the question... 

We should be using audio object for ondemand radio as they are audio clips of ondemand content.  
We use radioChannel on Live Radio pages as they are a Radio streams of channels. This was on the advice of SEO consultant.

Reply:
Should we add the publish date to the page's data or just have the upload date on the audioObject?

Reply:
Upload date is fine. We usually add publishedDate as part of the article schema.. we could add to the article object but it is just repeating upload date so not sure if it's adding any value https://schema.org/AudioObject